live-examples/js-examples/array/array-copywithin.html

```diff
@@ -1,11 +1,11 @@
 <pre>
 <code id="static-js">var array1 = ['a', 'b', 'c', 'd', 'e'];
 
-// copy to index 0 the element at index 3
+/* copy to index 0 the element at index 3 */
```

Comment:
@chharvey Can you change the comments back to the previous style since its single line comments